_pages/platform/algorithm-secrets.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: article
+title:  "Algorithm Secrets"
+excerpt: ""
+categories: basics
+nav_index: 40
+show_related: true
+image:
+    teaser: /icons/algo.svg
+permalink: /platform/algorithm-secrets/
+redirect_from:
+  - /basics/algorithm-secrets/
+---
+
+Algorithm secrets let you decouple sensitive data like passwords, access tokens, and credentials from your algorithm source code. With this feature, you can securely store secrets on Algorithmia and access them inside your algorithms using environment variables.
+
+#### Use Cases
+  - Connecting to a password-protected database
+  - Calling a secure HTTP webhook to `POST` your algorithm result
+
+Secrets are encrypted at rest, encrypted in transit, and only exposed to selected algorithms. We never expose sensitive data through the web interface or any user-facing APIs.
+
+Keep in mind that collaborators who can update algorithm source can also access its secrets.
+
+## Managing algorithm secrets
+
+Secrets are managed on the algorithm settings page. Here you'll be able to create, update, delete, and view your existing algorithm secrets. The steps below demonstrate the process for creating secrets.
+
+![Image of UI settings page for creating secrets](/developers/images/post_images/algorithm_secrets/settings_page.png)
+
+Start by clicking "New secret" to create an algorithm secret. Next, give your secret a name, description, key, and value. The key corresponds to the environment variable name and the value is the secret itself.
+
+It can take up to one minute for new secrets to show up inside your algorithm. Similarly, when you update a secret, it can take one minute for that change to take effect.
+{: .notice-info}
+
+![Image of UI create secret page](/developers/images/post_images/algorithm_secrets/create_secret_page.png)
+
+After your secret is configured, it can be updated or deleted on the settings page. Clicking "Update" will direct you to a form that is identical to the form shown above, and you'll be able to change the secret's name, description, and value.
+
+![Image of UI settings page for creating secrets](/developers/images/post_images/algorithm_secrets/settings_page_with_secret.png)
```

Comment:
This alt tag is identical to a previous one. I'd update to something that reflects that a secret exists on the page now.
```suggestion
![Image of UI settings page with an existing secret](/developers/images/post_images/algorithm_secrets/settings_page_with_secret.png)
```